lxd/devlxd.go

```diff
@@ -146,11 +146,39 @@ func devLXDAPIGetHandler(d *Daemon, r *http.Request) response.Response {
 		clientAuth = "trusted"
 	}
 
+	var supportedDrivers []api.DevLXDServerStorageDriverInfo
+
+	// Include supported storage drivers if the instance has the devLXD volume
+	// management security flag enabled.
+	if inst.ExpandedConfig()[string(devLXDSecurityMgmtVolumesKey)] == "true" {
+		url := api.NewURL().Path("1.0").URL
+		req, err := NewRequestWithContext(r.Context(), http.MethodGet, url.String(), nil, "")
+		if err != nil {
+			return response.DevLXDErrorResponse(err)
+		}
+
+		info := &api.Server{}
+
+		resp := api10Get(d, req)
```

Comment:
Rather than calling `api10Get` here (which gets a lot of other info unnecessarily here) and then discard all but the storage drivers info, please can you factor out the supported storage driver code from `api10Get` into its own function which you can call from `api10Get` and here.